libs/web/styles.css

```
@@ -78,6 +82,10 @@
     --accent-primary-1: #19bd82;
     --accent-primary-2: #015537;
 
+    --accent-secondary: #F95465;
+    --accent-secondary-1: #CA4757;
+    --accent-secondary-2: #552533;
+
     --accent-secondary: rgba(174, 237, 13, 0.4);
```

Biome (2.1.2)

[error] Duplicate custom properties can lead to unexpected behavior and may override previous declarations unintentionally.

--accent-secondary is already defined here.

Remove or rename the duplicate custom property to ensure consistent styling.

(lint/suspicious/noDuplicateCustomProperties)